### DISCUSSION TITLE
Passing Frigate+ API Key to Frigate LXC

### DISCUSSION THREAD

**Discussion by 3h50** [tteck/Proxmox#3819] [Help Needed]
Frigate wants the API key via docker compose.  How should this be done for the frigate LXC? https://deploy-preview-4055--frigate-docs.netlify.app/integrations/plus/#set-your-api-key

**Comment by wimb0** [comment]
I too, would like to know how to do this.
Created an issue for this: https://github.com/remz1337/Proxmox/issues/11

**Comment by SubNoizey** [comment]
keen to know how as well.

**Comment by carlosduque-incoxe** [comment]
Here is solved: (https://github.com/remz1337/Proxmox/issues/11#issuecomment-2411731581)

**Reply by tteck** [reply]
or here https://github.com/tteck/Proxmox/discussions/2711#discussioncomment-8975702

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/commit/5dbb000070c1062f502a7d4b17a70d2b4a6127fc
